### PR TITLE
CVE-2020-1711 init.

### PR DESCRIPTION
**Pull request by guitos**

### CODE REVIEW

File: 2020/1xxx/CVE-2020-1711.json [hunk 1]
```diff
@@ -4,15 +4,68 @@
...
+                {
+                    "vendor_name": "Red Hat",
+                    "product": {
+                        "product_data": [
+                            {
+                                "product_name": "QEMU",
+                                "version": {
+                                    "version_data": [
+                                        {
+                                            "version_value": "All qemu versions 2.x.x up to and including 2.12.0"
```
Comment (iamleot) [inline 2020/1xxx/CVE-2020-1711.json]: I think the version values are incorrect (probably due a confusion of Git version in the patch).

Can you please cite any references regarding that? (that versions 2.x.x up to and including 2.12.0 are affected?)

File: 2020/1xxx/CVE-2020-1711.json [hunk 2]
```diff
@@ -4,15 +4,68 @@
...
+                "refsource": "CONFIRM"
+            }
+        ]
     },
     "description": {
         "description_data": [
             {
                 "lang": "eng",
-                "value": "** RESERVED ** This candidate has been reserved by an organization or individual that will use it when announcing a new security problem. When the candidate has been publicized, the details for this candidate will be provided."
+                "value": "An out-of-bounds heap buffer access flaw was found in the way the iSCSI Block driver in QEMU versions 2.x.x up to and including 2.12.0 handled a response coming from an iSCSI server while checking the status of a Logical Address Block (LBA) in an iscsi_co_block_status() routine. A remote user could use this flaw to crash the QEMU process, resulting in a denial of service or potential execution of arbitrary code with privileges of the QEMU process on the host."
```
Comment (iamleot) [inline 2020/1xxx/CVE-2020-1711.json]: Please see the comment above regarding version (I think they are not correct).

File: 2020/1xxx/CVE-2020-1711.json [hunk 3]
```diff
@@ -4,15 +4,68 @@
...
+        ]
+    },
+    "references": {
+        "reference_data": [
+            {
+                "url": "https://bugzilla.redhat.com/show_bug.cgi?id=CVE-2020-1711",
+                "name": "https://bugzilla.redhat.com/show_bug.cgi?id=CVE-2020-1711",
+                "refsource": "CONFIRM"
+            }
+        ]
```
Comment (iamleot) [inline 2020/1xxx/CVE-2020-1711.json]: Please directly add corresponding upstream references:

 https://lists.gnu.org/archive/html/qemu-devel/2020-01/msg05535.html

...and email on oss-security@ ML:

 https://www.openwall.com/lists/oss-security/2020/01/23/3

**Comment by iamleot** [comment]
@cve-team, can we please wait for possible replies so it can be reviewed/adjusted?

File: 2020/1xxx/CVE-2020-1711.json (hunk 3 above)
Comment (guitos) [reply 2020/1xxx/CVE-2020-1711.json]: @iamleot,

I will check that, thanks.

**Comment by guitos** [comment]
@iamleot,

Please see https://github.com/CVEProject/cvelist/pull/3234.

**Comment by iamleot** [comment]
Hello Guilherme, 

Guilherme de Almeida Suckevicz writes:
> @iamleot,
>
> Please see https://github.com/CVEProject/cvelist/pull/3234.
>

Looks good, thank you very much for adjusting it!

**Pull request merged**